src/screens/WasteCollectionSettingsScreen.tsx

```diff
@@ -244,6 +256,10 @@ export const WasteCollectionSettingsScreen = () => {
   useEffect(() => {
     if (usedTypes) {
       dispatch({ type: WasteSettingsActions.setInitialWasteSettings, payload: usedTypeKeys });
+      // Activate notifications if the user has allowed system permissions
+      handleSystemPermissions().then((permission) => {
+        if (permission) dispatch({ type: WasteSettingsActions.toggleNotifications });
+      });
```

Comment:
Is the `permission` value here a boolean expression?

Reply:
yes

<img width="277" height="51" alt="image" src="https://github.com/user-attachments/assets/c4d0e50d-88b1-41cf-aa60-9e5303d20685" />